Source/Renderer/AutomaticUniforms.js

```diff
@@ -1484,6 +1484,70 @@ define([
             getValue : function(uniformState) {
                 return uniformState.shadowMap.shadowMapMatrix;
             }
+        }),
+
+        /**
+         * An automatic GLSL uniform representing the sun's shadow map cascade far splits.
+         *
+         * @private
+         *
+         * @alias czm_sunShadowMapCascadeSplitsNear
+         * @glslUniform
+         */
+        czm_sunShadowMapCascadeSplitsNear : new AutomaticUniform({
```

Comment:
We'll get better performance if we pack these uniforms into a single `mat4` uniform then just have GLSL helpers to "decompress" like we do for billboard vertex attributes.